packages/plugin-figma-blade-coverage/src/utils/sendAnalytics.ts

```diff
@@ -0,0 +1,40 @@
+import { incrementTotalUseCountAsync } from '@create-figma-plugin/utilities';
+
+export const sendAnalytics = async ({
+  eventName,
+  properties,
+}: {
+  eventName: string;
+  properties: object;
+}): Promise<void> => {
+  const pluginUsageCount = await incrementTotalUseCountAsync();
+  await fetch('https://api.segment.io/v1/track', {
+    method: 'POST',
+    headers: {
+      'content-type': 'application/json',
+      Authorization: `Basic ${process.env.SEGMENT_WRITE_KEY}`,
+    },
+    body: JSON.stringify({
+      userId: figma.currentUser?.id ?? `logged-out-user-${Date.now()}`,
```

Comment:
can logged out users even run the plugin?

Reply:
nupe. but because figma type for this is nullable and segment userId expects it to be string i had to do this